### PR TITLE
Fix: No device or entities

### PR DESCRIPTION
**Pull request by CM000n**
This should fix https://github.com/DurgNomis-drol/ha_toyota/issues/248
This should also give us more flexibility in future development, as we now accept all mytoyota versions >= 2.1.2,>3.0.

### CODE REVIEW

**Comment by TheNickFury** [comment]
Hi, is there any news on this change? I have a Toyota Hybrid if you need help..

**Comment by CM000n** [comment]
ping @DurgNomis-drol

**Comment by CM000n** [comment]
Ping @DurgNomis-drol 
Can you please approve this?
Thank you

File: custom_components/toyota/manifest.json [hunk 1]
```diff
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/DurgNomis-drol/ha_toyota",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/DurgNomis-drol/ha_toyota/issues",
-  "requirements": ["mytoyota==2.1.1", "arrow"],
+  "requirements": ["mytoyota>=2.1.2,<3.0", "arrow"],
```
Comment (Mrtenz) [inline custom_components/toyota/manifest.json:10]: `2.1.4` fixes a recent issue in the API.
```suggestion
  "requirements": ["mytoyota>=2.1.4,<3.0", "arrow"],
```

File: pyproject.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.11"
 homeassistant = "^2023.11"
-mytoyota = "2.1.1"
+mytoyota = "^2.1.2"
```
Comment (Mrtenz) [inline pyproject.toml:11]:
```suggestion
mytoyota = "^2.1.4"

```

**Comment by CM000n** [comment]
Already resolved by a duplicated Merge Request https://github.com/DurgNomis-drol/ha_toyota/pull/289

**Pull request closed without merging**